unstructured/ingest/cli/interfaces.py

```diff
@@ -59,6 +61,45 @@ def add_cli_options(cmd: click.Command) -> None:
         pass
 
 
+class CliRetryStrategyConfig(RetryStrategyConfig, CliMixin):
+    @staticmethod
+    def add_cli_options(cmd: click.Command) -> None:
+        options = [
+            click.Option(
+                ["--max-tries"],
```

Comment:
can we can call this `max-retries` as you had it before? I think this makes it clear it's tied to retry logic